toml/src/main/kotlin/org/rust/toml/crates/local/CrateResolverService.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.crates.local
+
+import com.intellij.openapi.components.service
+
+interface CrateResolverService {
```

Comment:
Originally I thought that we also might implement this interface with the crates.io resolver. Is that futile?

Reply:
Frankly, I'm not sure if this would be very useful in general since using crates.io API is slow and isn't suitable enough for real-time inspections in IDE. But if you know cases when such implementation would be nice to have, please feel free to describe them

Reply:
I only wanted to do that to make the inspections work if the local (git) is not available for some reason. But I agree that its functionality (and latency) leaves a lot to be desired. I'll rename the service :)